concordium-contracts-common/src/types.rs

```diff
@@ -20,6 +21,12 @@ pub type HashMap<K, V, S = fnv::FnvBuildHasher> = hashbrown::HashMap<K, V, S>;
 /// the `fnv` hash function.
 pub type HashSet<K, S = fnv::FnvBuildHasher> = hashbrown::HashSet<K, S>;
 
+/// Contract index.
+pub type ContractIndex = u64;
+
+/// Contract subindex.
+pub type ContractSubIndex = u64;
```

Comment:
It would be more useful to provide a bit of context as to what these indices mean. The documentation that is currently added is not useful.

Reply:
I have added a bit more now